README.md

```diff
@@ -67,6 +67,20 @@ See [examples](./examples/README.md)
 - @influxdata/influxdb-client-apis
   - [setup / onboarding](./examples/onboarding.js)
   - [health](./examples/health.js)
+  
+#### InfluxDB 1.8 API compatibility
+
+InfluxDB 1.8.0 introduced forward compatibility APIs for InfluxDB 2.0. This allow you to easily move from InfluxDB 1.x to InfluxDB 2.0 Cloud or open source.
```

Comment:
We should better use influxdb documentation that went through proofreading, so it could be as simple as:

[InfluxDB 2.0 API compatibility endpoints](https://docs.influxdata.com/influxdb/v1.8/tools/api/#influxdb-2.0-compatibility-endpoints) are part of the InfluxDB 1.x line since InfluxDB 1.8.0.
This allows you to leverage InfluxDB 2.0 client libraries for both writing and querying data with Flux. For more details, see

- [InfluxDB 1.8 example](examples/influxdb-1.8.ts)
- https://docs.influxdata.com/influxdb/v1.8/about_the_project/releasenotes-changelog/#forward-compatibility